optimum/exporters/openvino/model_configs.py

```diff
@@ -1808,7 +1808,8 @@ def __init__(
         height: int = DEFAULT_DUMMY_SHAPES["height"],
         **kwargs,
     ):
-        super().__init__(task, normalized_config, batch_size, num_channels, width, height, **kwargs)
+        # Reduce img shape by 4 for FLUX to reduce memory usage on conversion
+        super().__init__(task, normalized_config, batch_size, num_channels, width // 4, height // 4, **kwargs)
```

Comment:
Cool ! This will also override the user given values (from cli or python api), I would prefer changing these values in the function signature defaults.